doc/en/assertions.rst

```diff
@@ -696,6 +696,46 @@ however this can be overridden by explicitly passing ``category`` argument while
               result.equal(i * 2, i * 2, category='Multiples')
 
 
+This schema highlights the structure of a summarised output
+
+    .. code-block:: bash
```

Comment:
Pretty minor but can you change to `code-block:: none`? This disables syntax highlighting, since the code isn't actually bash you don't want to highlight keywords like `for`, the `|` symbol etc.